modules/standard/Error.chpl

```diff
@@ -41,13 +41,13 @@ use SysBasic;
 
 // here's what we need from Sys
 pragma "no doc"
-extern proc sys_strerror_syserr_str(error:syserr, out err_in_strerror:err_t):c_string;
+private extern proc sys_strerror_syserr_str(error:syserr, out err_in_strerror:err_t):c_string;
 
 /* This function takes in a string and returns it in double-quotes,
    with internal double-quotes escaped with backslash.
    */
 pragma "no doc"
-proc quote_string(s:string, len:ssize_t) {
+private proc quote_string(s:string, len:ssize_t) {
```

Comment:
I'm not sure about this one. I think quote_string really should be in a different module, have a better name, and not take in a length (since a Chapel string has a length). But I wouldn't try to do that until Kyle's work merges...